### ISSUE TITLE
[Adaptive Cards-Save File]: Focus order is not logical on the application on after closing the 'Save File' dialog through Esc key.

### ISSUE THREAD

**Issue by Rishikesh6521** [microsoft/AdaptiveCards#9212]
### Target Platforms

UWP

### SDK Version

1.0.3.0

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Application: Adaptive Cards Visualizer
OS Version: 25H2 (OS Build 26200.5670) 
App Version: 1.0.3.0

**Pre-requisites:**
Navigate using keyboard only.

**Repro steps:**
1. Open the Adaptive Cards Visualizer application.
2. Navigate to the 'Save File' control present on the header section using tab key and activate using enter key.
3. Now 'Save File' dialog appears and close the dialog using esc key.
4. Verify that, focus order is logical on the page on after closing the 'Save File' dialog through Esc key or not.

**Actual Result:**
Focus order is not logical on the application on after closing the 'Save File' dialog through Esc key. Currently on after closing the dialog through esc key focus gets lost and next Tab key press focus lands on the 'New Card' control and on pressing three Tab key stokes focus lands on the 'Save File' control and same way in by pressing seven times Shift+Tab key strokes focus lands on the 'Save File' control due to which users face difficulty on again navigating to the triggered control.

**Expected Result:**
Focus order should be logical on the application on after closing the 'Save File' dialog through Esc key. After closing the 'Save File' dialog through esc key focus should not get lost and it should directly return back to the same triggered control i.e., 'Save File' control.

**User Impact:**
Keyboard dependent users will be impacted if the focus order is not logical on the application on after closing the 'Save File' dialog through Esc key. Due to which users will not be able to perform the required action as it initiates the incorrect flow and user again need to perform extra keystrokes navigate to the required control and might get confused in terms of identifying the focus order of the application.

**WCAG Reference link:**
https://www.w3.org/WAI/WCAG22/Understanding/focus-order.html

Have feedback to share on Bugs? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/user-attachments/assets/ae25c3b1-b5d1-40b7-9b53-13eb7278e0f5

### Card JSON

```JSON
N/A
```
